sonaranalyzer-dotnet/its/regression-test.ps1

```diff
@@ -241,9 +387,14 @@ try {
 
     Write-Header "Processing analyzer results"
 
+    CheckInternalProjectsDifferences
+
     Write-Host "Normalizing the SARIF reports"
     $sarifTimer = [system.diagnostics.stopwatch]::StartNew()
-    Exec { & .\create-issue-reports.ps1 }
+    
+    # Normalize & overwrite all *.json SARIF files found under the "actual" folder
```

Comment:
The code was moved `create-issue-reports.ps1`.  We now import the file and use it as a library instead of executing it. This is old behavior and I prefer to keep also the comment.